include/boost/sml.hpp

```diff
@@ -207,6 +207,16 @@ struct remove_reference<T &&> {
 };
 template <class T>
 using remove_reference_t = typename remove_reference<T>::type;
+template <class T>
```

Comment:
Can we change files inside boost folder instead of sml.hpp and generate sml.hpp with tools/pph.sh instead

Reply:
I have fixed it. Check the new commit please.